pyccl/tests/test_tkkssc.py

```diff
@@ -249,3 +249,146 @@ def test_tkkssc_counterterms_gc(kwargs):
+                                     'is_number_counts2': True,
+                                     'is_number_counts3': False,
+                                     'is_number_counts4': True},
+                                    {'is_number_counts1': True,
+                                     'is_number_counts2': False,
+                                     'is_number_counts3': False,
+                                     'is_number_counts4': True},
+                                    {'is_number_counts1': True,
+                                     'is_number_counts2': False,
+                                     'is_number_counts3': False,
+                                     'is_number_counts4': False},
+                                    {'is_number_counts1': True,
+                                     'is_number_counts2': False,
+                                     'is_number_counts3': True,
+                                     'is_number_counts4': False},
+                                    {'is_number_counts1': False,
+                                     'is_number_counts2': False,
+                                     'is_number_counts3': True,
+                                     'is_number_counts4': False},
+                                    ])
+def test_tkkssc_linear_bias(kwargs):
+    hmc = ccl.halos.HMCalculator(COSMO, HMF, HBF, mass_def=M200,
+                                 nlog10M=2)
+    k_arr = KK
+    a_arr = np.array([0.3, 0.5, 0.7, 1.0])
+
+    # Tk's exact version
+    prof = ccl.halos.HaloProfileNFW(ccl.halos.ConcentrationDuffy08(M200),
+                                    fourier_analytic=True)
+    # Error when prof is not NFW
+    with pytest.raises(TypeError):
+        tkk_lin = ccl.halos.halomod_Tk3D_SSC_linear_bias(COSMO, hmc, prof=P2)
+
+    bias1 = 1
+    bias2 = 2
+    bias3 = 3
+    bias4 = 4
+    is_nc = False
+
+    # Tk's from tkkssc_linear
+    tkk_lin = ccl.halos.halomod_Tk3D_SSC_linear_bias(COSMO, hmc, prof=prof,
+                                                     bias1=bias1,
+                                                     bias2=bias2,
+                                                     bias3=bias3,
+                                                     bias4=bias4,
+                                                     is_number_counts1=is_nc,
+                                                     is_number_counts2=is_nc,
+                                                     is_number_counts3=is_nc,
+                                                     is_number_counts4=is_nc,
+                                                     lk_arr=np.log(k_arr),
+                                                     a_arr=a_arr)
+    _, _, _, tkk_lin_arrs = tkk_lin.get_spline_arrays()
+    tk_lin_12, tk_lin_34 = tkk_lin_arrs
+    # Remove the biases
+    tk_lin_12 /= (bias1 * bias2)
+    tk_lin_34 /= (bias3 * bias4)
+    assert np.abs(tk_lin_12 / tk_lin_34 - 1).max() < 1e-5
```

Comment:
why are you testing this for all combinations of kwargs? If this doesn't depend on kwargs, put it in a separate test.

Reply:
Because this test is quite fast and that way I didn't have to write a different test. Even if I split the test I still need to compute `tk_lin_12` and `tk_lin_34` to subtract it below. I'll leave it as is unless you still want me to put it in a different test.